benchexec/tools/wasp-c.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    This class serves as tool adaptor for WASP-C (https://github.com/wasp-platform/wasp)
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable('wasp-c', subdir='bin')
+
+    def working_directory(self, executable):
+        executableDir = os.path.dirname(executable)
+        return executableDir
```

Comment:
In general, I advise on writing tools in a way such that they work no matter from which directory they are called. This is much more convenient for users, less error-prone, etc. If your tool supports this, we can remove this method. Is this the case?

Reply:
Yes it does, I'll remove the method.